app/queries/entity_connections_query.rb

```diff
@@ -38,6 +38,6 @@ def run
       .where(@category_id ? "link.category_id = #{@category_id}" : nil)
       .order(link_count: :desc)
       .page(@page)
-      .per(PER_PAGE)
+      .per(@per_page || PER_PAGE)
```

Comment:
thanks for catching this. Because @per_page is [defined during `initialize`](https://github.com/public-accountability/littlesis-rails/blob/c1bb6601c26f7bc1b3b37ae43bf320841302572a/app/queries/entity_connections_query.rb#L10) this could be just @per_page.